index.d.ts

```diff
@@ -275,6 +271,14 @@ declare module '@primer/components' {
     label?: string
   }
 
+  export interface TruncateProps extends StyledSystem.MaxWidthProps, TextProps {
```

Comment:
This should extend `Typography` props and `BaseProps` versus `TextProps` (the TextProps is referring to the `Text` component)

Reply:
Good catch! Thank you!